app/celer-sim/simple-driver.py

```diff
@@ -91,6 +94,7 @@ def strtobool(text):
     'num_track_slots': num_tracks,
     'max_steps': max_steps,
     'initializer_capacity': 100 * max([num_tracks, num_primaries]),
+    'optical_buffer_capacity': 100 * max([num_tracks, num_primaries]),
```

Comment:
```suggestion
    'optical_buffer_capacity': 3 * max_steps * num_tracks,
```
Does the test still fail if you increase this buffer size so it's large enough to hold all the distribution data? Since we don't actually convert the distributions into optical photons yet the buffers will just keep growing, and in addition to the failures you mention I am seeing:
```console
/home/alund/celeritas_project/celeritas/app/celer-sim/celer-sim.cc:251: critical: While running input at <stdin>: celeritas: runtime error: insufficient capacity (4500) for buffered scintillation distribution data (total capacity requirement of 4521)
/home/alund/celeritas_project/celeritas/src/celeritas/optical/detail/ScintPreGenAction.cc:84: 'buffer_size + core_state.size() <= buffer.size()' failed
```

Looks like you'll also need to make sure `simple_calo` is disabled for the lar-sphere test.